Localide/Classes/Localide.swift

```diff
@@ -40,7 +40,7 @@ public final class Localide: NSObject {
      - Apple Maps
      - Installed 3rd party apps which are supported by Localide and included in the QuerySchemes
     */
-    public lazy var availableMapApps: [LocalideMapApp] = Localide.installedMapApps()
+    public var availableMapApps: [LocalideMapApp] { Localide.installedMapApps() }
```

Comment:
Old Localide problem where it is not aware that app ares uninstalled because it gets the list on first use only.

Now that it's a computed property it checks every time. Manually tested this.

This change also resolves the problem we saw where CoPilot is no longer installed, but it's still selected in Bringg's App Settings.